test/unit/gateways/paymill_test.rb

```diff
@@ -225,6 +251,10 @@ def test_transcript_scrubbing
 
   private
 
+  def store_endpoint(credit_card, currency, amount)
```

Comment:
Why is this endpoint needed? It looks like there were already unit tests set up to test the `store` action, so why is this needed now?

Reply:
This is a method that generates the endpoint URL that the ```store``` operation makes its request to. It's used by the new tests added here. I named the method as a noun for consistency with the existing helper methods like ```successful_store_response```, but it could also be named something like ```generate_store_endpoint```. I'll go with ```store_endpoint_url``` since that seems like it would be more clear.

Reply:
Nice, thanks!